### ISSUE TITLE
Update frame documentation

### ISSUE THREAD

**Issue by tmorrell** [caltechlibrary/dataset#103]
Current frame commands in the documentation are outdated. The below python snippets can probably be used:

```
# frame_refresh() updates the objects in a data frame based on
# the current state of the collection. Any objects removed from
# the collection will be removed from the frame.
#
# Args: collection_name (string), frame_name (string), keys (JSON source)

# frame_reframe() replaces the object list in a data frame.
#
# Args: collection_name (string), frame_name (string), keys (JSON source)

# frame_clear() removes all objects from a frame.
#
# Args: collection_name (string), frame_name (string)
```

**Comment by rsdoiel** [comment]
Talked it over with Tom, refresh_frame will no longer take a list of keys. It only operates on the data frames existing key list (minus any deleted keys in the collection).  On the otherhand frame_reframe() will regenerate the from using a new key list.
So this is a functional change as well as a document bug.

**Comment by rsdoiel** [comment]
The code changes in frames.go and libdataset.go were already done. The comments were out of date so I updated the comments to conform to how the code is currently written.  Will merge issue-103 branch when testing and updating docs is completed.

**Comment by rsdoiel** [comment]
This is fixed in release v0.1.9.
